v2/cli/term/page_router.go

```diff
@@ -63,13 +64,15 @@ func (r *pageRouter) loadEventPage(eventID string) {
 	r.loadPage(eventPageName, func() {
 		r.eventPage.refresh(eventID)
 	})
+	go r.eventPage.streamEventLog(eventID)
```

Comment:
We definitely shouldn't start streaming these logs until the user actually elects to view them.

Reply:
I'm getting an issue where if I start the goroutine after the user hits "L", it doesn't show logs until I pull up the logs modal again. Is it okay to load logs upon loading the event page, or is this a separate issue?

Reply:
Actually it's even weirder, I can type any key and it would refresh and print out all the correct logs. I wonder what could be causing this

Reply:
Don't worry about log streaming yet. Just use static placeholder text for now.